IdokladSdk.IntegrationTests/Tests/Clients/ProformaInvoice/ProformaInvoiceTests.Async.cs

```diff
@@ -132,6 +133,37 @@ public async Task DeleteAsync_SuccessfullyDeleted()
             Assert.IsTrue(data);
         }
 
+        [Test]
+        public async Task AccountMultipleProformasAsync_SuccessfullyAccounted()
+        {
+            // Arrange
+            var proformaModel = (await _proformaInvoiceClient.DefaultAsync()).AssertResult();
+            proformaModel.PartnerId = PartnerId;
+            proformaModel.Description = "Invoice";
+            proformaModel.DateOfPayment = DateTime.UtcNow.SetKindUtc();
+            proformaModel.Items.Clear();
+            proformaModel.Items.Add(new ProformaInvoiceItemPostModel
+            {
+                Name = "Test",
+                UnitPrice = 100,
+                Amount = 1
+            });
+            var proformaId1 = (await _proformaInvoiceClient.PostAsync(proformaModel)).AssertResult().Id;
+            var proformaId2 = (await _proformaInvoiceClient.PostAsync(proformaModel)).AssertResult().Id;
+            var putModel = new AccountProformaInvoicesPutModel { ProformaIds = new[] { proformaId1, proformaId2 } };
+
+            // Act
+            var result = (await _proformaInvoiceClient.AccountMultipleProformaInvoicesAsync(putModel)).AssertResult();
+
+            // Assert
+            Assert.NotNull(result);
+
+            // Teardown
+            await _issuedInvoiceClient.DeleteAsync(result.Id);
```

Comment:
To bych vyřešil TearDown metodou. Když spadne assert, tak se Ti to sem nedostane. Stejně tak  v testu ne-async metody.